tests/engineer/test_engineer.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+import numpy as np
+import xarray as xr
+
+from src.engineer import Engineer
+
+from ..utils import _make_dataset
+
+
+class TestEngineer:
+
+    def _setup(self, data_path):
+        # setup
+        interim_folder = data_path / 'interim'
+        interim_folder.mkdir()
+
+        expected_output, expected_vars = [], []
+        for var in ['a', 'b']:
+            (interim_folder / f'{var}_preprocessed').mkdir()
+
+            # this file should be captured
+            data, _, _ = _make_dataset((10, 10), var)
+            filename = interim_folder / f'{var}_preprocessed/hello.nc'
+            data.to_netcdf(filename)
+
+            expected_output.append(filename)
+            expected_vars.append(var)
+
+            # this file should not
+            (interim_folder / f'{var}_preprocessed/boo').touch()
+
+        # none of this should be captured
```

Comment:
so what is this testing?

Reply:
This function just sets up stuff (since it's name doesn't start with `test`, no tests are run on it). It's used by all the other tests.